client/lib/screen-title/index.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Internal dependencies
+ */
+import { setTitle as legacySetTitle } from './actions';
+
+/**
+ * Given a Redux store instance, subscribes to the store, updating the page
+ * when the title state changes. Currently, this dispatches through the legacy
+ * Flux actions used elsewhere in the codebase.
+ *
+ * @param {Object} store Redux store instance
+ */
+export function subscribeToStore( store ) {
+	let title = store.getState().documentSettings.title;
```

Comment:
>Fine by me, thought using a reserved word might've not been acceptable.

Hmm, I'm curious now, but I don't think it'd be a problem if it's called as a property of the state object. Might be a problem if someone was to try to destructure the object, i.e.

```js
const { document } = store.getState();
```